v4/source/introduction.yaml

```diff
@@ -214,23 +214,193 @@ tags:
 
       The list of Mattermost WebSocket events are:
       - added_to_team
+        ```
+        {
+          one: {
+            id: t('combined_system_message.added_to_team.one'),
+            defaultMessage: '{firstUser} **added to the team** by {actor}.',
+          },
+          one_you: {
+            id: t('combined_system_message.added_to_team.one_you'),
+            defaultMessage: 'You were **added to the team** by {actor}.',
+          },
+          two: {
+            id: t('combined_system_message.added_to_team.two'),
+            defaultMessage: '{firstUser} and {secondUser} **added to the team** by {actor}.',
+          },
+          many_expanded: {
+            id: t('combined_system_message.added_to_team.many_expanded'),
+            defaultMessage: '{users} and {lastUser} were **added to the team** by {actor}.',
+          }
```

Comment:
Is this supposed to be the contents of the WebSocket event? Because I'm pretty sure that's not what is sent with this event